boards.txt

```diff
@@ -2119,6 +2119,94 @@ heltec_wifi_lora_32.menu.UploadSpeed.512000.upload.speed=512000
 
 ##############################################################
 
+heltec_wifi_lora_32_V2.name=Heltec_WIFI_LoRa_32_V2
+
+heltec_wifi_lora_32_V2.upload.tool=esptool_py
+heltec_wifi_lora_32_V2.upload.maximum_size=1310720
+heltec_wifi_lora_32_V2.upload.maximum_data_size=327680
+heltec_wifi_lora_32_V2.upload.wait_for_upload_port=true
+
+heltec_wifi_lora_32_V2.serial.disableDTR=true
+heltec_wifi_lora_32_V2.serial.disableRTS=true
+
+heltec_wifi_lora_32_V2.build.mcu=esp32
+heltec_wifi_lora_32_V2.build.core=esp32
+heltec_wifi_lora_32_V2.build.variant=heltec_wifi_lora_32_V2
+heltec_wifi_lora_32_V2.build.board=Heltec_WIFI_LoRa_32_V2
+
+heltec_wifi_lora_32_V2.build.f_cpu=240000000L
+heltec_wifi_lora_32_V2.build.flash_mode=dio
+heltec_wifi_lora_32_V2.build.flash_size=4MB
```

Comment:
This should be 8MB to cover the flash on the V2 HW.

Reply:
Thanks for your notice @n9wxu.

@me-no-dev I had submisson a new PR [#2474](https://github.com/espressif/arduino-esp32/pull/2474), it's include a new partition map as @n9wxu suggested. This one is OK and I will close PR [#2470](https://github.com/espressif/arduino-esp32/pull/2470)

I think [#2407](https://github.com/espressif/arduino-esp32/pull/2407) can be closed too.

Reply:
Looks good.